torch/_dynamo/config.py

```diff
@@ -62,7 +62,7 @@
 # with assume_static_by_default=True.
 # With this flag enabled, we always compile a frame as fully static for the first time, and, if we fail
 # any guards due to wobbles in shape, we recompile with *all* the wobbled shapes as being marked dynamic.
-automatic_dynamic_shapes = True
+automatic_dynamic_shapes = False
```

Comment:
You can't do this without simultaneously fixing `dynamic=True` behavior, or simultaneously flipping the default on `dynamic_shapes`. By changing this only, you have literally disabled dynamic shapes UNLESS this config flag was explicitly modified.

I still think you need to finish https://github.com/pytorch/pytorch/pull/99469 first, you're just digging yourself deeper in a hole here.